pkg/lib/interaction/nodes/create_authenticator_whatsapp_otp.go

```diff
@@ -29,6 +30,12 @@ func (e *EdgeCreateAuthenticatorWhatsappOTP) Instantiate(ctx *interaction.Contex
 	if err != nil {
 		return nil, err
 	}
+	err = ctx.Events.DispatchEvent(&nonblocking.WhatsappOTPVerifiedEventPayload{
```

Comment:
If we add the DispatchEvent call in WhatsappCodeProvider.VerifyCode, it would be less likely we will forget to DispatchEvent in the future, when we have an additional place to call WhatsappCodeProvider.VerifyCode